### PR TITLE
inbox and messageboard update

### PR DESCRIPTION
**Pull request by SalleeMatthew**
This update gives the security chief the ability to view and modify all inbox and messageboards.

### CODE REVIEW

File: api/src/repositories/messageboard/messageboard.repository.ts [hunk 1]
```diff
@@ -36,11 +36,20 @@ export class MessageboardRepository {
       )
       .from<Member, Member[]>('member')
       .where('id', memberId);
+    const securityinfo = await knex
+      .select(
+        'member_id',
+      )
+      .from<RoleAssignment, RoleAssignment[]>('role_assignment')
+      .where('role_id', '62')
```
Comment (smile0711) [inline api/src/repositories/messageboard/messageboard.repository.ts:44]: We cannot put an id number in directly as your id for security chief maybe different than mine or ctr's.  We fix this by using a rolemap.  If you look at colony services, you will see an example.

Comment (SalleeMatthew) [reply api/src/repositories/messageboard/messageboard.repository.ts]: Changed it to where it is using rolemap as requested.

File: api/src/controllers/inbox.controller.ts [hunk 2]
```diff
@@ -22,6 +22,11 @@ class InboxController {
   }
   
   public async adminCheck(placeId, id, type): Promise<any> {
+    try {
```
Comment (smile0711) [inline api/src/controllers/inbox.controller.ts:25]: Can we add an if typeof type is undefined for all of these new tries?  The reason is if type is colony, hood, or block then the getAdminInfo will run twice.

Comment (SalleeMatthew) [reply api/src/controllers/inbox.controller.ts]: Fixed and ready for review.

**Pull request closed without merging**